cmd/doctor.go

```diff
@@ -495,3 +501,93 @@ func runDoctorScriptType(ctx *cli.Context) ([]string, error) {
 	}
 	return []string{fmt.Sprintf("ScriptType %s is on the current PATH at %s", setting.ScriptType, path)}, nil
 }
+
+func runDoctorCheckDBConsistency(ctx *cli.Context) ([]string, error) {
+	_, committer, err := models.TxDBContext()
```

Comment:
How about create a function in `models` package?

Reply:
I think this is the way to go ...

Reply:
... every thing else makes it more complicated